examples/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/README.md

```diff
@@ -0,0 +1,41 @@
+# quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration
+
+
+
+## Overview
+
+![simple-quickstart-arch](https://user-images.githubusercontent.com/5663078/229119386-0dbc6e30-a060-465e-86dd-f89712b0fc49.png)
+
+The quickstart template enables you to begin working with your Machine Learning models rapidly, using your MongoDB Atlas Cluster and Amazon SageMaker Endpoint as tools. With this template, you can utilize MongoDB as a data source and Amazon SageMaker for data analysis, streamlining the process of building and deploying machine learning models.
+
+
+
+## MongoDB Atlas CFN Resources used by the templates
+
+- [MongoDB::Atlas::Trigger](../../cfn-resources/trigger/)
```

Comment:
Is this really the only MongoDB Atlas CFN resource used by this template?

Reply:
yes, the template asks the user to provide ProjectId and collection and DB name. You can search for `MongoDB::Atlas::` in the [template](https://github.com/aws-quickstart/quickstart-mongodb-atlas-analytics-amazon-sagemaker-integration/blob/0ca645280a474cad82443caa3bddb1c6b6c71e39/templates/mongodb-sagemaker-analytics.template.yaml#L98)